cub/benchmarks/bench/merge/keys.cu

```diff
@@ -0,0 +1,165 @@
+// SPDX-FileCopyrightText: Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
```

Comment:
```suggestion
// SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION. All rights reserved.
```
Applies 2 more times

Reply:
I yet have to get used to the fact that we arrived in 2025...